frontend/src/components/StaticMap/StaticMap.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useMemo, useRef, useState } from 'react'
+import { generateSize, getScaleTranslation } from './StaticMap.utils'
+import { geoMercator, geoPath } from 'd3-geo'
+import { useTheme } from '@mui/system'
+import Box from '@mui/material/Box'
+import PropTypes from 'prop-types'
+
+const StaticMap = ({ geoJSON, square, height: mapHeight, areaId }) => {
```

Comment:
There's a way to use an SVG's `viewBox` and `preserveAspectRatio` to skip the need to define its size or worry about resize events, but this is performing well enough that I'd say let's not bother trying that unless we run into performance problems.